wordpress-mobile/WordPress-Android
# Jetpack App (Basics): No Jetpack Sites Empty View

Parent #14302 

This PR implements the empty state UI for logging into the Jetpack app when the account has no jetpack sites.

Implementation Details
- Two large vector assets were added, don't let the number of lines cause panic. :)
- The colors in the `img_jetpack_empty_state` were changed to semantic where possible. I considered adding custom colors to the color file, but since they are not common I decide to leave them in the drawable itself. This follows the same pattern as the other illustrations. I'm opening to changing this if need be.
- The word `Error` was dropped from the fragment, viewmodel and navigationevent.  I was on the fence about including `Jetpack` and would love thoughts on whether it should be included. Also all files were left in WordPress source sets. This can be moved and would love opinions. 
- Refactored the way binding was being handled in `LoginNoSitesFragment`
- Removed the errorMessage from the `ShowNoJetpackSites` navigation event since it never changes and is set in the layout xml

To test:
- Download this branch and build a Jetpack App variant
- Tap the 'Login with WordPress' button
- Enter WordPress.com credentials for an account that is not linked to any Jetpack sites
- After logging in you should be brought to the No Jetpack Sites Found empty view

## Regression Notes
1. Potential unintended areas of impact 🟢 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and automated tests

3. What automated tests I added (or what prevented me from doing so)
Added new unit tests in `LoginNoSitesViewModelTest`

PR submission checklist:
